### DISCUSSION TITLE
Unable to remove the notification dot cuz of a deleted repo

### DISCUSSION THREAD

**Discussion by qwedc001** [community/community#82108] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Well,there was a repository I once forked and proposed an issue,it seems that the author repiled,but then the repo was deleted due to I don't know exactly why,maybe violation of the github actions' rules?
the thing is,It now remains an issue that I can't get the blue dot of the deleted repo out of my notification box,making me check the notifications every time I come here.That's kinda disturbing.

**Comment by faridnec** [comment]
If you have notifications related to a repository that no longer exists (due to deletion or any other reason), and you want to clear those notifications, you can try this method - I done this before -
after open notification, scroll down to find the notification related to the deleted repository. You can use the "Mark all as read" button if you have many notifications to clear.
If you are still getting notifications for the deleted repository, it's possible that you are subscribed to the thread -> Open the issue or pull request related to the deleted repository -> In the right sidebar, look for the "Unsubscribe" button and click it.

**Reply by qwedc001** [reply]
still having issues,
I can't really see the notification itself,together with the mark as read.
My situation is attached below.
![QQ截图20231227121638](https://github.com/community/community/assets/14366166/eac922af-dda9-47b1-b456-95eb8c544f30)

**Comment by nguyenducc** [comment]
It seems like you're facing an issue with a GitHub repository. If I understand correctly, you forked a repository, proposed an issue, and the author replied. However, the repository was later deleted, possibly due to a violation of GitHub Actions' rules. Now, you're stuck with a notification for the deleted repository, and the issue remains in your notification box.

To resolve this issue, you can try the following steps:

1. **Mark as Read:**
   - Go to your GitHub notifications.
   - Find the notification related to the deleted repository.
   - Mark it as "Read" to clear the blue dot.

2. **Unwatch the Repository:**
   - Navigate to the original repository or the issue page.
   - If you are "Watching" the repository, click on the "Unwatch" button to stop receiving notifications for it.

3. **Check Notification Settings:**
   - Go to your GitHub account settings.
   - Navigate to "Notification Center" or a similar section.
   - Adjust your notification settings to your preference.

If the issue persists, you might want to reach out to GitHub Support for further assistance, as they can provide more specific guidance based on the current status of the repository and your account settings.

**Reply by qwedc001** [reply]
your understanding is correct,but unfortunately none of these worked.Further information is pasted on the previous answer's reply.
Thanks for your assistance,I'll try to contact github support.

**Comment by qwedc001** [comment]
solved by contacting github support.

**Reply by JohnLGalt** [reply]
Just ran into this - contacted Support to get it fixed, thanks for replying back with this.

**Comment by ascpixi** [comment, marked as answer]
For anyone that has this issue - after contacting GitHub Support, they gave me the following steps to take in order to fix this:
- create a personal access token with the `notifications` scope: https://github.com/settings/tokens/new
- run these commands in your shell of choice:
```sh
export now=$(date +%Y-%m-%dT%H:%M:%SZ)
export TOKEN=<INSERT GITHUB PERSONAL ACCESS TOKEN HERE>
curl -H "Authorization: bearer $TOKEN" -X PUT -H "Accept: application/vnd.github.v3+json" https://api.github.com/notifications -d '{"last_read_at":"'"$now"'"}'
```

Worked perfectly for me to get rid of a stuck notification for a DMCA'd repository! Hope this helps! ☆*: .｡. o(≧▽≦)o .｡.:*☆

**Reply by katrinabrock** [reply]
@qwedc001 if you're still around can you mark this one as selected answer?

**Reply by winniehell** [reply]
thanks for posting this here! :heart:

easier way if you have the [GitHub CLI](https://cli.github.com/) installed:

```shell
gh api -X PUT /notifications
```

**Reply by Magicdover** [reply]
Thanks! That’s such an easy way to solve the problem.

**Reply by Opisek** [reply]
"gitcointeam" still shows up in the "repositories" section in my inbox. How can I get rid of it?

**Reply by jtlicardo** [reply]
> thanks for posting this here! ❤️
> 
> easier way if you have the [GitHub CLI](https://cli.github.com/) installed:
> 
> ```shell
> gh api -X PUT /notifications
> ```

Life saver

**Reply by wd** [reply]
The repo is still in the repo list after you marked it as read. Please don't mark it as read. You need to delete it. 

Use `gh api notifications` to get the notification id, and use `gh api --method DELETE /notifications/threads/<id>` to delete it. 

If you marked it as read already, it will not show in the results of `gh api notifications`. You will need to do another workaround. 

Select the repo and post below in the browser console. (I don't know which one is required.)
```
document.querySelector('.js-notifications-mark-all-actions').removeAttribute('hidden'); 
document.querySelector('.js-notifications-mark-all-actions form[action="/notifications/beta/archive"] button').removeAttribute('disabled'); 
```
Then you will be able to click the `Mark as unread` button. The notification will be shown in the results of `gh api notifications` now. 
<img width="285" height="160" alt="image" src="https://github.com/user-attachments/assets/654b6f80-1201-46c8-a8fd-ceafddc2c491" />

**Reply by pyk** [reply]
thanks @wd , I use the following one liner to clear the stale spam github notifications:

```
gh api notifications --jq '.[].id' | while read id; do gh api --method DELETE /notifications/threads/"$id" --silent; done
```

**Reply by doxxx** [reply]
@wd Thanks for the detailed instructions, you saved me from my OCD :)
